### ISSUE TITLE
Suggestion - Allow for custom loot tables

### ISSUE THREAD

**Issue by DWX12** [thraaawn/RiddleChests#16]
So far the kind of loot I've seen are gems such as emerald and diamonds, ender pearls, and gear that is diamond and/or enchanted.

Nothing is picked up from various other mods nor is anything injected into riddle chests. There might be cases where players want to replace vanilla chests with riddle ones, but keep the same loot table, or that players may want to create their own loot table for a custom pack.

Basically just looking for more loot options.
